app/pages/summary/contextCreator.test.js

```diff
@@ -56,6 +63,16 @@ describe('order summary contextCreator', () => {
       expect(context.dateSummaryCreated).toEqual('19 July 2020');
     });
 
+    it('should return the dateCompletedLabel for a completed order', () => {
+      const context = getContext({ orderId: 'order-1', orderData: mockCompletedOrderData });
+      expect(context.dateCompletedLabel).toEqual(completeManifest.dateCompletedLabel);
+    });
+
+    it('should return the mockCompletedOrderData as the current date for a completed order', () => {
```

Comment:
think this test description is bit misleading. It's not the currentDate. It'll just be the date that is provided in mockCompletedOrderData

Reply:
hm yeah will change that